tests/scripts/thread-cert/Cert_5_3_07_DuplicateAddress.py

```diff
@@ -117,7 +117,13 @@ def test(self):
         self.nodes[ED2].add_ipaddr('2001::1')
         time.sleep(3)
 
-        self.nodes[ED3].ping('2001::1')
+
+        # Final ping doesn't need a response;
+        # used to trigger an ADDR_ERR.ntf
+        try:
+            self.nodes[ED3].ping('2001::1')
+        except pexpect.TIMEOUT:
```

Comment:
@darconeous @jwhui 
Given that this test is one of the only to excercise manual address configuration, and the NoRoute error idicates some issue with the on-mesh prefix configuration, I looked through the log again. It looks like earlier in the test, the insert prefix command is failing:
```
prefix add 2001::/64 paros med
### ==> 0x5A = SPINEL_PROP_THREAD_ON_MESH_NETS
### ==> add prefix 2001::/64,  stable=1, flags=0x73
TX Pay: (22) ['81', '04', '5A', '20', '01', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '40', '01', '73'] 
RX Pay: (91) ['80', '06', '73', '54', '00', '60', '00', '00', '00', '00', '2C', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'B4', '40', 'D4', '8C', 'E9', '38', 'F9', '52', 'FF', '02', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '02', '4D', '4C', '4D', '4C', '00', '2C', 'B6', '50', '00', '15', '00', '00', '00', '00', '00', '00', '00', '00', '01', '44', '76', 'D6', 'FD', '36', '6F', 'EA', 'EA', 'CC', 'BC', 'F5', '02', 'A3', '01', 'D9', '97', '20', 'D6', 'CB', 'E0', '92', 'B1', 'C4', 'B3', 'FA', 'FE', '32'] 
RX Pay: (122) ['80', '06', '73', '73', '00', '60', '00', '00', '00', '00', '4B', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'B4', '40', 'D4', '8C', 'E9', '38', 'F9', '52', 'FE', '80', '00', '00', '00', '00', '00', '00', 'D0', '8D', '7F', '87', '58', '88', 'FC', 'CB', '4D', '4C', '4D', '4C', '00', '4B', '27', '06', '00', '15', '01', '00', '00', '00', '00', '00', '00', '00', '01', 'EF', 'F7', '65', '51', 'E5', 'D4', '88', '89', '96', 'D1', '76', '1F', '16', '7A', '0E', '36', '01', '75', 'A7', '36', 'CB', '32', 'B9', 'EE', '94', 'B3', 'DD', '8D', 'B0', '1D', '6A', '22', 'E2', '76', '14', '59', 'B9', '34', 'B3', '80', '9C', '14', 'BB', 'FB', 'B7', '9C', '08', '7B', '58', '74', '36', 'D8', 'DC', '18', '49', '83', '09', '52'] 
RX Pay: (105) ['80', '06', '73', '62', '00', '60', '00', '00', '00', '00', '3A', '11', 'FF', 'FE', '80', '00', '00', '00', '00', '00', '00', 'B4', '40', 'D4', '8C', 'E9', '38', 'F9', '52', 'FE', '80', '00', '00', '00', '00', '00', '00', 'D0', '8D', '7F', '87', '58', '88', 'FC', 'CB', '4D', '4C', '4D', '4C', '00', '3A', '55', '9A', '00', '15', '02', '00', '00', '00', '00', '00', '00', '00', '01', '00', 'C8', '07', 'CD', 'C3', '87', '62', '71', '5C', '83', '36', '51', '75', '49', 'F4', '38', '17', '07', '33', '54', '3E', '14', '69', 'F8', '26', 'EE', 'E4', '08', '85', '42', 'A0', '2E', '3A', '2E', '18', 'F9', 'FE', '20', 'CF', '5E', '5D'] 
RX Pay: (6) ['81', '06', '00', '04', 'F6', '5D'] 
PROP_VALUE_IS: LAST_STATUS = 4
STATUS_INVALID_STATE : The given operation is invalid for the current state of the device.
Done
```